### ISSUE TITLE
Translucent snow/rain

### ISSUE THREAD

**Issue by Wundcreme** [X0nk/Bliss-Shader#340]
### Describe the bug
Rain, especially snow, is transparent and hard to recognize.

### Describe how your minecraft is set up
![2024-11-12_12 59 53](https://github.com/user-attachments/assets/5e8c9924-93cd-4d07-b044-fa5cbdcf76af)

### Describe how to reproduce the bug
Go into a snow biome and make it rain.

![2024-11-12_12 56 21](https://github.com/user-attachments/assets/a5543e81-ed12-4bf0-b781-5d61a57c2b11)

**Comment by X0nk** [comment]
yeah not really a bug - more so just a problem i have to solve. as an intentional choice, i made rain/snow exist purely as the "bloomy fog" effect. adjusting this effect will cause rain/snow to be more or less visible.
i have changed how bloom is recently, which negatively impacted the bloomy fog effect. im working on bringing back how it used to look while not ruining how the new bloom looks
